Platforms/QemuQ35Pkg/ConfigKnobs/ConfigDataGfx/ConfigDataGfx.c

```diff
@@ -64,16 +64,23 @@ ApplyGfxConfigToPolicy (
   GfxEnablePort0   = *(BOOLEAN *)ConfigBuffer;
   GfxSiliconPolicy = AllocateCopyPool (sizeof (DefaultQemuGfxPolicy), DefaultQemuGfxPolicy);
 
+  if (GfxSiliconPolicy == NULL) {
+    DEBUG ((DEBUG_ERROR, "Failed to allocate Policy structure\n"));
+    goto Exit;
+  }  // Mu
+ 
   // We only translate the GFX ports #0 exposed to platform from conf data
   GfxSiliconPolicy[0].Power_State_Port = GfxEnablePort0;
 
   Status = SetPolicy (&gPolicyDataGFXGuid, POLICY_ATTRIBUTE_FINALIZED, GfxSiliconPolicy, sizeof (DefaultQemuGfxPolicy));
+
   if (EFI_ERROR (Status)) {
     DEBUG ((DEBUG_ERROR, "%a Failed to update GFX policy per configuration data - %r!!!\n", __func__, Status));
     ASSERT (FALSE);

```

Comment:
Not related to your change, but I suggest updating this assert so it is more useful when printed. `ASSERT (FALSE)` is not very helpful.

```suggestion
    ASSERT_EFI_ERROR (Status);
```

Reply:
Good suggestion. Updated w/ recommendations.